### PR TITLE
Create api/resources route

### PR DESCRIPTION
**Pull request by minwwgo**
create api  for resources.

### CODE REVIEW

**Review by ferhatu — APPROVED** [review]

**Review by ahmad-ali14 — CHANGES_REQUESTED** [review]
please ignore package-lock.json

File: server/server.js [hunk 1]
```diff
@@ -1,13 +1,36 @@
...
+const mongoose=require('mongoose')
+require("dotenv").config();
+const bodyParser = require("body-parser");
 
 const app = express();
 app.use(cors());
+app.use(morgan('dev'))
+
+const resources = require("./routes/resources");
+const db = process.env.MONGO_URL
```
Comment (ahmad-ali14) [inline server/server.js]: all imports should be on the top

File: server/server.js [hunk 2]
```diff
@@ -20,10 +43,11 @@ if (process.env.NODE_ENV === 'production') {
         res.sendFile(path.join(__dirname, 'client', 'build', 'index.html'));
     });
 }
+// Use Router for each api
+app.use("/api/resources", resources);
```
Comment (ahmad-ali14) [inline server/server.js]: put this one after line 36

File: server/server.js (hunk 1 above)
Comment (minwwgo) [reply server/server.js]: I moved to top .

File: server/server.js (hunk 2 above)
Comment (minwwgo) [reply server/server.js]: Done move to line 38

**Review by ahmad-ali14 — APPROVED** [review]

**Pull request merged**